style.css

```diff
@@ -0,0 +1,98 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+body {
+  font-size: 1.4rem;
+  line-height: 1.3;
+  text-align: center;
+  color: black;
+  margin: 0;
+
+  /*   text-align: center */
+}
+
+/* #main{
+  background-color: #e9ecef;
+  margin: 35px 9px;
+  border-radius: 10px; */
```

Comment:
- [ ] Great job on spotting this detail. But kindly ensure that you always add a why to your required change so the reviewee feels more open to acknowledging and accepting it. Here you might say something like "Kindly remove all unused code from your stylesheet since it causes clutter in the code and reduces code maintainability."